contracts/ERC20Snapshot.sol

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: MIT
```

Comment:
is it necessary to pull this in?

Reply:
It would show a solidity warning, so its better to add it. And if you want to just ignore the solidity warning you can do `// SPDX-License-Identifier: UNLICENSED`